### PR TITLE
2.x support for rxjava2 with 1.6 support, (dont merge)

### PR DESCRIPTION
**Pull request by jutoft**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,3 +1,5 @@
+group = 'io.reactivex.rxswing2'
```
Comment (akarnokd) [inline build.gradle]: Could you update this to 'io.reactivex.rxjava2' as it is the new "standard" parent project for 2.x - based components. The full id will then be `io.reactivex.rxjava2:rxswing:2.0.0-RC1`

**Comment by akarnokd** [comment]
I've attempted a change as well but run into trouble.

- First of all this should target branch 2.x but there is none yet.
- The nebula plugin rejects version changes greater than 1, i.e., 0.27.0 -> 2.0.0 crashes the gradle build very early on
- I'm not in favor of many design decisions RxSwing took.

**Comment by jutoft** [comment]
Yes, i can not push 2.x, someone would have to pull it and push it from my branch.
The library is still small so changing the design could be done, what is your suggestion @akarnokd ?

**Comment by akarnokd** [comment]
> Yes, i can not push 2.x, someone would have to pull it and push it from my branch.

This can only work if there is an 1.x branch created **and** released so git has a 1.x.y tag in its history. Then a new branch out of 1.x can be named 2.x and can be replaced with v2 code.

> The library is still small so changing the design could be done, what is your suggestion

I'm still evaluating the design options.

**Pull request closed without merging**